runtime/rasdump/dmpagent.c

```diff
@@ -431,13 +431,13 @@ static const J9RASdumpSpec rasDumpSpecs[] =
 		NULL,
 		NULL,
 		doJavaVMExit,
-		{ 0,
+		{ J9RAS_DUMP_ON_GP_FAULT | J9RAS_DUMP_ON_ABORT_SIGNAL,
```

Comment:
Why would exit need to trigger on gp or abort?

Reply:
A zero `eventMask` seems not to be expected in some parts of the code I looked at.
@theresa-m is that the motivation for this change? If so, why not also at line 418?

Reply:
J9RAS_DUMP_ON_GP_FAULT or J9RAS_DUMP_ON_ABORT_SIGNAL are needed to pass this `if` statement https://github.com/theresa-m/openj9/blob/master/runtime/rasdump/trigger.c#L558 so the dump agent can acquire exclusive vm access.

Reply:
The body of that `if` statement will only execute if _none_ of `J9RAS_DUMP_ON_GP_FAULT`, `J9RAS_DUMP_ON_ABORT_SIGNAL` or `J9RAS_DUMP_ON_TRACE_ASSERT` are present, and it can only _gain_ exclusive access _within_ that `if` statement.

Reply:
ah yes you are right. this line of flags is not needed.